src/WpfMath/Boxes/OverUnderBox.cs

```diff
@@ -3,24 +3,24 @@
 
 namespace WpfMath.Boxes
 {
-    // Box representing other box with delimeter and script box over or under it.
+    // Box representing other box with delimiter and script box over or under it.
     internal class OverUnderBox : Box
     {
-        public OverUnderBox(Box baseBox, Box delimeterBox, Box scriptBox, double kern, bool over)
+        public OverUnderBox(Box baseBox, Box delimiterBox, Box scriptBox, double kern, bool over)
             : base()
         {
             this.BaseBox = baseBox;
-            this.DelimeterBox = delimeterBox;
+            this.DelimiterBox = delimiterBox;
             this.ScriptBox = scriptBox;
             this.Kern = kern;
             this.Over = over;
 
             // Calculate dimensions of box.
             this.Width = baseBox.Width;
-            this.Height = baseBox.Height + (over ? delimeterBox.Width : 0) +
+            this.Height = baseBox.Height + (over ? delimiterBox.Width : 0) +
                 (over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0);
-            this.Depth = baseBox.Depth + (over ? 0 : delimeterBox.Width) +
-                (!over && scriptBox == null ? 0 : scriptBox.Height + scriptBox.Depth + kern);
+            this.Depth = baseBox.Depth + (over ? 0 : delimiterBox.Width) +
+                (!over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0.0);
```

Comment:
For consistency.

```suggestion
                (!over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0);
```

Reply:
Usually I prefer the other way around, but that's a good suggestion. I'll change every `0` into `0.0` in this block :3